server/sonar-server/src/test/java/org/sonar/server/user/UserUpdaterTest.java

```diff
@@ -79,6 +78,9 @@
   @Mock
   NewUserNotifier newUserNotifier;
 
+  @Mock
```

Comment:
I didn't say to not mock, just don't use annotation with the Mockito runner. Declare mocks using Mockito.mock(…) method